tools/scripts/tidy.toml

```diff
@@ -4,7 +4,7 @@
 
 # This is a cargo-make file included in the toplevel Makefile.toml
 
-[tasks.tidy]
+[tasks.tidy-minus-fmt]
```

Comment:
Suggestion (optional): Delete the "tidy-minus-fmt" task, move these up into "ci-job-tidy", and make "tidy" invoke fmt + ci-job-tidy.

Reply:
I kinda want the ci-job-foo to be toplevel so you can change CI jobs without changing other stuff.

Reply:
eh, fixed it anyway